tasks_week3_Israel_Jacob/include/Dog.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "Animal.h"
+class Dog : public Animal
+{
+private:
+	int steps;
+public:
+	Dog(std::string name);
+	void printDetails() const override;
+	char getInitial() const override;
+	void move() override;
+	void step() override;
```

Comment:
no use of virtual 
check this
https://stackoverflow.com/questions/2391679/why-do-we-need-virtual-functions-in-c

Reply:
Got changed